packages/apollo-server-core/src/__tests__/logger.test.ts

```diff
@@ -1,5 +1,5 @@
 import { ApolloServerBase } from '../..';
```

Comment:
You've moved this test to `@apollo/utils.logger`, so I'd think you'd be able to delete it, as well as the top-level dependencies on the various logging packages.

Reply:
Decided to replace with tests that ensure the logger is correctly used by AS / passed through the request pipeline, but yes - removed the other logger deps! e5ebcff